<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ventana Principal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblNombre">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombre">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGuardar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtRaza">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblRaza">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Raza</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbEdad">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblEdad">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Edad</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>444</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>txtNombre</tabstop>
  <tabstop>txtRaza</tabstop>
  <tabstop>sbEdad</tabstop>
  <tabstop>btnGuardar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
